lib/index.js

```diff
@@ -77,6 +77,28 @@ class Client {
     return ActionApi.action(this.api, this.apiKey, actionType, data)
   }
 
+  /**
+   * Parameters for get package request
+   * @param {string} id - Id or name of the package
+   * @param {boolean} useDefaultSchema - Indicates whether to use default package schema instead of a custom schema defined with an IDatasetForm plugin (default: false)
+   * @param {boolean} includeTracking - Indicates whether to add tracking information to dataset and resources (default: False)
+   */
+  async retrieve(id, useDefaultSchema = false, includeTracking = false) {
```

Comment:
We don't need `useDefaultSchema` and `includingTracking` as options - just use the default valus in the call. If people want that detail they can use action API. Keep this as simple as possible.

for `id` i would name variable as `nameOrId`